doc/prop.xml

```diff
@@ -8,6 +8,106 @@
 #############################################################################
 ##
 
+<#GAPDoc Label="IsPartialOrderDigraph">
+<ManSection>
+  <Prop Name="IsPartialOrderDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    This is a synonym for <Ref Prop="IsReflexiveDigraph"/> and
+    <Ref Prop="IsAntisymmetricDigraph"/> and <Ref Prop="IsTransitiveDigraph"/>.
+
+    <Example><![CDATA[
+gap> gr := Digraph([[1, 3], [2, 3], [3]]);
+<digraph with 3 vertices, 5 edges>
+gap> IsPartialOrderDigraph(gr);
+true
+gap> gr := CycleDigraph(5);
+gap> IsPartialOrderDigraph(gr);
+false
+]]></Example>
+  </Description>
+</ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="IsMeetSemilatticeDigraph">
+<ManSection>
+  <Prop Name="IsMeetSemilatticeDigraph" Arg="digraph"/>
+  <Prop Name="IsJoinSemilatticeDigraph" Arg="digraph"/>
+  <Prop Name="IsLatticeDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    
+    <C>IsMeetSemilatticeDigraph</C> returns <K>true</K> if the digraph
+    <A>digraph</A> is a meet semilattice; <C>IsJoinSemilatticeDigraph</C>
+    returns <K>true</K> if the digraph <A>digraph</A> is a join semilattice;
+    and  <C>IsLatticeDigraph</C> returns <K>true</K> if the digraph
+    <A>digraph</A> is both a meet and a join semilattice.
+    <P/>
+
+    A digraph is a <E>meet semilattice</E> if it is a partial order and every
+    pair of vertices has a unique greatest lower bound (meet) with respect to
+    that partial order. A <E>join semilattice</E> is a partial order where
+    every pair of vertices has a unique least upper bound (join).
+
+    <Example><![CDATA[
+gap> gr := Digraph([[1, 3], [2, 3], [3]]);
+<digraph with 3 vertices, 5 edges>
+gap> IsMeetSemilatticeDigraph(gr);
+true
+gap> IsJoinSemilatticeDigraph(gr);
+false
+gap> IsLatticeDigraph(gr);
+false
+gap> gr := Digraph([[1], [2], [1 .. 3]]);
+<digraph with 3 vertices, 5 edges>
+gap> IsJoinSemilatticeDigraph(gr);
+true
+gap> IsMeetSemilatticeDigraph(gr);
+false
+gap> IsLatticeDigraph(gr);
+false
+gap> gr := Digraph([[1 .. 4], [2, 4], [3 ,4], [4]]);
```

Comment:
Again, this should have an output, saying `<digraph with 4 vertices, 9 edges>`.
Also, fix the spacing in `[3 ,4]` to `[3, 4]` to fit in with the rest of the package.